tasks/collector.py

```diff
@@ -36,6 +59,76 @@
 }
 
 
+class YAMLValidationError(Exception):
+    def __init__(self, message):
+        super().__init__(message)
+
+
+def validate_manifest(manifest):  # return list of components to remove, empty list if valid
+    # validate collector version matches ocb version
+    version_checks = ["otelcol_version"]
```

Comment:
I originally thought we needed to check the "version" parameter originally, but reading the ocb documentation that parameter is just the version of the collector being built, not specifying the collector version used or component version used. Can change it to one variable, but thought it might be useful as a list if we need to add more version parameter checks in the future

Reply:
removed the list and simplified the code.